CODING.md

```diff
@@ -180,6 +180,25 @@ We do not have strong policies against repetition in test code, as usual the imp
 
 We write them using [spread](https://github.com/snapcore/spread). Generally all externally visible features and behaviour should have spread tests. It's also important to have them for robustness and fallback behaviour as related to system properties.
 
+In order to keep the integration testing harness easy to read and consistent, there are some rules about the order and the existence of the different spread tests sections.
+
+This is the ordered sections list to follow:
+  . summary (required)
+  . details (required)
+  . backends
+  . systems
+  . manual
+  . priority
+  . warn-timeout
+  . kill-timeout
+  . environment
+  . prepare
+  . restore
+  . debug
+  . execute (required)
```

Comment:
Are these supposed to be bullet points with `*` or are they intentionally inline? Just pointing it out in case it wasn't intentional

Reply:
improved to a ordered list